bdsf/spectralindex.py

```diff
@@ -60,7 +60,7 @@ def __call__(self, img):
                 unav_freqs = freqin[good_chans]
                 nmax_to_avg = img.opts.specind_maxchan
                 nchan = unav_image.shape[0]
-                mylog.info('After initial flagging of channels by rms, %i good channels remain' % (nchan,))
+                mylogger.userinfo(mylog, 'Number of channels remaining after initial flagging', str(nchan))
```

Comment:
Curious as to why you changed this from `mylog.info` to `mylogger.userinfo`.

Reply:
That was so that the message is printed to the terminal and not only to the log file.